src/demo-app/examples/card/fancy/card-fancy-example.scss

```diff
@@ -0,0 +1,8 @@
+md-card {
```

Comment:
Can you make any styles applied to the component done via css class that is prefixed with `example-`? In material 1, people often had difficulty telling the difference between the example styles and the component styles.